code/class-textdex.php

```diff
@@ -371,16 +375,14 @@ public function update( array $order_ids ) {
 		}
 
 		if ( $this->is_ready() ) {
-			/*
-			Do this all at once to avoid autocommit overhead. */
+			// Do this all at once to avoid autocommit overhead.
 			// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery,WordPress.DB.DirectDatabaseQuery.NoCaching
 			$wpdb->query( 'BEGIN;' );
 			foreach ( $order_ids as $order_id ) {
-				/*
-				Get rid of old metadata */
+				// Get rid of old metadata
```

Comment:
380 | ERROR | Comment closer must be on a new line
     |       | (Squiz.Commenting.BlockComment.CloserSameLine)